gp-includes/routes/translation.php

```diff
@@ -315,6 +317,12 @@ public function translations_post( $project_path, $locale_slug, $translation_set
 				$translations = GP::$translation->for_translation( $project, $translation_set, 'no-limit', array('translation_id' => $translation->id), array() );
 
 				if ( ! empty( $translations ) ) {
+					if ( $glossary ) {
+						require_once GP_TMPL_PATH . 'helper-functions.php';
```

Comment:
I think it should be a template function as it's creating html for the front end to use.  A custom template might want to change it from a tooltip to something else and need different markup to do it.

I'd say if someone has created a new theme and template functions that it would have to support all functions in the helper file to properly support outputting the glossary markup.  

The other way to do this, which might be logically cleaner, would be to create a new template "translation-row-ajax.php" which would call `map_glossary_entries_to_translations_originals()` from inside of it to keep all of the template logic inside the template directory.